src/shared/hooks/use-context-init-message.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext, useContext } from "react";
+import { IInitInteractive } from "@concord-consortium/lara-interactive-api";
+
+export const InitMessage = createContext<IInitInteractive | null>(null);
```

Comment:
I would name this `InitMessageContext` to make the distinction between the message and the context that wraps it clear, and then client code would look like:
```
      <InitMessageContext.Provider value={initMessage}>
        { renderMode() }
      </InitMessageContext.Provider>
```

Reply:
@kswenson Thanks. Makes sense to me.